at-tanvo/Exercise1.rb

```diff
@@ -0,0 +1,11 @@
+def non_reapeating
+  a = "asic"
+  arr = a.split('')
+  chua = []
+  arr.each do |n|
+    chua.push(arr.index(n)) if arr.count(n) == 1
+  end
+  return chua[0] if chua[0]
+  -1
+end 
+puts non_reapeating()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/47f87600d63703e4dcdf4c1477df3685adc982eb/at-tanvo/Exercise1.rb#L11"></span>Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens)</td>
    </tr>
  </tbody>
</table>